syntax = "proto2";

package teststruct; // Replace with your actual package name

message testStruct {
  required int32 IntValue = 1;
  required string CharValue = 2;
  required float FloatValue = 3;
}
